blogposts/2024-05-15-improved-gallery-fabric-0.74.md

```diff
@@ -0,0 +1,102 @@
+
+Get React Native Gallery from the [Microsoft store]((https://www.microsoft.com/store/productId/9NPG0B292H4R?ocid=pdpshare)) or get the latest changes from our (Github repository)[https://github.com/microsoft/react-native-gallery]!
+
+![rn-gallery-gif-2024](D:\react-native-devblog\blogposts\assets\gallery-demo-2024.gif)
+
+# 🗺️ Roadmap to Fabric
+
+Introducing [Fabric](https://reactnative.dev/architecture/fabric-renderer), the latest rendering system for React Native. Designed for efficiency across platforms in C++, RNW Fabric seamlessly targets Composition while allowing hosting XAML islands within. Fabric-powered apps default to Win32 architecture, aligning with WinAppSDK and WinUI3 standards. Stay tuned for migration guidance for current UWP RNW apps. Follow our journey to Fabric [here](https://github.com/microsoft/react-native-windows/issues/12042).
+
+*I think it would be super slay to have a demo of an app here. Some choices include Gallery on Fabric (hopefully updated to Chris's latest changes), Playground RNTester, ChatAI App*
+
+## Increasing Core Functionality
+
+We have been hard at work getting component parity up to speed with the older Paper rendering system. Several components, including Views, TextInput, and ActivityIndicator, already support a large set of properties and methods, with more components in the work. Here's a current snapshot of our progress:
+
+| Priority | Component | Available | Properties |
+| --- | --- | --- | --- | 
+| 0 | View| ✅ | 90% | 
+| 0 | Text| ✅ | 85% | 
+| 0 | Image| ✅ | 89% | 
+| 0 | TextInput| ✅ | 80% | 
+| 1 | ScrollView| ✅ | 84% | 
+| 1 | Modal| 🟥 | 64% | 
+| 2 | ActivityIndicator| ✅ | 86% | 
+| 2 | Switch| ✅ | 90% | 
+| 2 | RefreshControl| 🟥 | 28% | 
+*As of February 2024*
+
+## Getting Community Modules Onboard
+
+*Do we have a plan for this?*
+
+## Current Timeline
+
+*Copied mostly from the Github issue, need to know what the current expected timeline is*
+
+At this time, Fabric on Windows isn't quite ready for widespread adoption. While we're actively working to enhance both functionality and the developer experience, there are still significant gaps to address. If you're an early adopter who doesn't mind diving into an evolving landscape, you can explore Fabric [here](https://github.com/microsoft/react-native-windows/wiki/Using-the-new-architecture-templates). Keep in mind that documentation may be limited, and we recommend reviewing the provided lists of yet-to-be-implemented features before submitting any bug reports. Rest assured, as we continue our journey, support for Fabric will steadily expand.
```

Comment:
We can save the call to action (or warning 😅) for a future blog post. For now we want to inform the broad strategy. People can click through to the pinned issue if they want more.